scania-digital-design-system/tegel
# feat(footer): add compact footer property

## **Describe pull-request**  
Adding "CompactDesign" property to make footer half size.

## **Issue Linking:**  
_Choose one of the following options_
  https://github.com/scania-digital-design-system/tegel/issues/650

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to stroybook
2. Check it in footer component

## **Checklist before submission**

## **Suggested test steps**
- [ x] Browser testing (Chrome, Safari, Firefox) 

## **Screenshots**  
<img width="1501" alt="image" src="https://github.com/scania-digital-design-system/tegel/assets/8792533/82d582d6-c7c2-4eca-a8e9-bd56714754c4">


## **Additional context**  
_Add any other context or feedback requests about the pull-request here._
